cigaradvisor/blocks/article-list/article-list.js

```diff
@@ -39,9 +39,15 @@ export async function renderPage(wrapper, articles, limit) {
     return articleTeaser;
   });
 
+  let counter = 0;// counter for images to be loaded eagerly
   await Promise.all(articlePromises).then((results) => {
     results.forEach((teaser) => {
       list.append(teaser);
+      if (counter < 2) {
+        const articleTeaserImage = teaser.querySelector('.article-image img');
+        articleTeaserImage.setAttribute('loading', 'eager');
+        counter += 1;
+      }
     });
   });
   list.querySelector('.article-teaser.block .article-image img')?.setAttribute('loading', 'eager');
```

Comment:
Rather than use the counter, use this query selector to grab the first two results. 

I also think we should move this to line 133 or so, so that we only apply the eager loading to the first page of the list, rather than the first two on every page.

Reply:
done